lib/Jsrt/ChakraCommon.h

```diff
@@ -996,7 +996,7 @@ typedef unsigned short uint16_t;
     /// </returns>
     CHAKRA_API
         JsSetCurrentContext(
-            _In_ JsContextRef context);
+            _In_opt_ JsContextRef context);
```

Comment:
You'll need to update the annotation in Jsrt.cpp:

https://github.com/Microsoft/ChakraCore/blob/6543dfb6a478d6a5f8aece6285ba12566c634797/lib/Jsrt/Jsrt.cpp#L793

Reply:
Thanks. I pushed a commit with the change.